adagucserverEC/CDrawImage.cpp

```diff
@@ -1426,11 +1426,15 @@ int CDrawImage::createGDPalette(CServerConfig::XMLE_Legend *legend) {
   return 1;
 }
 
-void CDrawImage::rectangle(int x1, int y1, int x2, int y2, CColor innercolor, CColor outercolor) {
+void CDrawImage::rectangle(int x1, int y1, int x2, int y2, CColor innercolor, CColor outercolor, bool noVerticalSpace) {
```

Comment:
I think it would be better to do this calculation in where you call rectangle.

It should be possible to achieve what you want without touching the draw functions.

Reply:
Maybe I can play with the size of the block to be one pixel longer in height

Reply:
Yes, it was easier to fix than I thought. You can have a look again.